### ISSUE TITLE
help requested: postfix-(send|receive)-access.cf not found

### ISSUE THREAD

**Issue by pn2** [tomav/docker-mailserver#847]
I am struggling configuring this image. 

I got IMAP access to the server working, using let's encrypt certificates extracted from Traefik. However, mail delivery does not work. This seems to be related to the files 'postfix-send-access.cf' and 'postfix-receive-access.cf' not existing, but I don't understand what I need to change to have them created.

In the container, /var/log/syslog shows:
```
Feb 17 20:57:51 mail postfix/smtpd[3676]: error: open database /tmp/docker-mailserver/postfix-send-access.cf: No such file or directory
Feb 17 20:57:51 mail postfix/smtpd[3676]: error: open database /tmp/docker-mailserver/postfix-receive-access.cf: No such file or directory
```
An except of the messages during container startup with the debug option turned on is attached (domain.tld of course is my own domain). 
[container_messages.txt](https://github.com/tomav/docker-mailserver/files/1734176/container_messages.txt)
 
Thanks for your help

**Comment by 17Halbe** [comment]
see https://github.com/tomav/docker-mailserver/issues/844#issuecomment-366208862

PR is waiting for being accepted

**Comment by pn2** [comment]
Thats solved it, thanks!

**Comment by johansmitsnl** [comment]
Should be fixed in: #846
